src/hooks/useUsers.js

```diff
@@ -0,0 +1,26 @@
+import {useEffect, useState} from "react";
+
+const useUsers = () => {
+  const [users, setUsers] = useState([]);
+  const [message, setMessage] = useState("");
```

Comment:
i like the fact that you are doing error handling here! its always important to think about "what should i display if things go wrong?" because things will go wrong all the time haha